blocks/footer/footer.css

```diff
@@ -1,14 +1,267 @@
-footer {
-  padding: 2rem;
-  background-color: var(--light-color);
-  font-size: var(--body-font-size-s);
+footer .footer-wrapper {
+  background: #fcfcfc;
 }
 
-footer .footer {
-  max-width: 1200px;
+footer .section.footer-top-note {
+  padding-top: 0;
+  padding-bottom: 1.25rem;
+}
+
+.footer-links .section-container{
```

Comment:
please prepend all footer related css with .block.footer to keep it in footer scope

Reply:
the indentation seems a bit messed.

Reply:
agreed,done